src/cascadia/TerminalSettingsEditor/GlobalAppearance.cpp

```diff
@@ -5,16 +5,55 @@
 #include "GlobalAppearance.h"
 #include "GlobalAppearance.g.cpp"
 #include "MainPage.h"
+#include "EnumEntry.h"
+
+#include <LibraryResources.h>
 
 using namespace winrt;
 using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Xaml::Controls;
 using namespace winrt::Microsoft::Terminal::Settings::Model;
+using namespace winrt::Windows::Foundation::Collections;
 
 namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 {
-    GlobalAppearance::GlobalAppearance()
+    GlobalAppearance::GlobalAppearance() :
+        _ElementThemes{ winrt::single_threaded_observable_vector<Editor::EnumEntry>() }
     {
         InitializeComponent();
+
+        auto elementThemeMap = EnumMappings::ElementTheme();
+        for (auto [key, value] : elementThemeMap)
+        {
+            // Uppercase the first letter to conform to our current Resource keys
+            std::wstring_view enumName = key;
+            auto fmtKey = fmt::format(L"Globals_Theme{}{}/Content", char(std::towupper(enumName[0])), enumName.substr(1));
+            auto entry = winrt::make<EnumEntry>(GetLibraryResourceString(fmtKey), winrt::box_value<ElementTheme>(value));
+            _ElementThemes.Append(entry);
+

```

Comment:
is it possible for this to live in EnumMappings? Like, we just call EnumMappings::ElementTheme() and it prepares a vector of entries with the right names?

Reply:
Otherwise, we'll start to scatter this format string generation stuff all across the codebase.

Reply:
can TSM reach across to TSE's resources file?

Reply:
**OH**, this is in TSM. Interesting . . .

Reply:
I don't super love that. Huh. I wonder what we can do ...

Maybe there's space in TSE/Utils for a helper? `std::wstring LocalizedNameForEnumName(wstring_view category, wstring_view type, wstring_view valuename)`?